api/orchestrator/orchestrator.proto

```diff
@@ -470,7 +470,10 @@ message StoreAssessmentResultRequest {
   clouditor.assessment.v1.AssessmentResult result = 1
       [ (validate.rules).message.required = true ];
 }
-message StoreAssessmentResultResponse {
+
+message StoreAssessmentResultResponse {}
```

Comment:
It would be good to have a small comment here, why this is empty and why the ResultsResponse is not empt